app/views/clients/cas_readiness/_manual.haml

```diff
@@ -47,8 +47,8 @@
             .mr-10
               = f.input :disability_verified_on, as: :boolean, label: GrdaWarehouse::Hud::Client.cas_columns[:disability_verified_on], input_html: {checked: @client.disability_verified_on.present?}
             .mr-10.pt-2
-              - tags = GrdaWarehouse::AvailableFileTag.tag_includes('Verification of Disability').map(&:name)
-              - file = @client.client_files.tagged_with(tags, any: true).order(:created_at).last
+              - tags = @client.cas_attributes_file_tag_map[:disability_verified_on]
+              - file = @client.client_files.tagged_with(tags, any: true).order(updated_at: :desc).first
```

Comment:
@eanders I changed the sort order here to be consistent with https://github.com/greenriver/hmis-warehouse/blob/5a0b49fcaa76359b65ee88b752c67d0092c01784/app/models/concerns/cas_client_data.rb#L263-L265 since that's what's actually used to set the value.

Reply:
that sounds like the right sort order.  Good catch.